sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -24,6 +24,8 @@ import java.util.regex.Pattern
 
 import scala.collection.mutable.ArrayBuffer
 
+import org.apache.commons.codec.binary.{Base64 => CommonsBase64}
```

Comment:
does `Base64` conflict with anything?

Reply:
seems like so, otherwise we won't use full class name in the code.

Reply:
Yep, it conflicts with the expression itself since it's also named `Base64`.